data/sidebar_docs.yaml

```diff
@@ -1,78 +1,82 @@
-- title_en: Robonomics Home Assistant Integration
+- title_en: Smart Home
   items:
     - title_en: Overview
       link: /docs/robonomics-smart-home-overview
       withoutNav: true
-    - title_en: Pre-installed Image For Raspberry Pi
+    - title_en: Install Smart Home
+      link: /docs/install-smart-home
```

Comment:
since you've renamed the article, fix link in the end of **ROBONOMICS SMART HOME OVERVIEW** (**START HERE YOUR SMART HOME** section, **FOR NEW USERS** button which leads to `/docs/hass-image-install/`)